src/supabase/supabase_manager.py

```
@@ -587,6 +587,326 @@ def upload_all(self, tables: List[str] = None,
+        Raises:
+            RuntimeError: If no management API token or if execution fails
+        """
+        import requests
+        token = self._get_management_api_token()
+        if not token:
+            raise RuntimeError(
+                "No management_api_token in config/config_local.ini — "
+                "required for sync-schema DDL operations"
+            )
+        ref = self._get_project_ref()
+        resp = requests.post(
```

Ruff (0.15.2)

[error] Probable use of `requests` call without timeout

(S113)